sources/Outputs/src/Job.cpp

```diff
@@ -51,7 +51,7 @@ const std::string Job::solverFilename_ = "solver.par";
 const std::string Job::solverName_ = "dynawo_SolverSIM";
 const std::string Job::solverParId_ = "SimplifiedSolver";
 
-Job::Job(const JobDefinition& def) : def_{def} {}
+Job::Job(JobDefinition&& def) : def_{std::forward<JobDefinition>(def)} {}
```

Comment:
Why is this needed? or is it just for the beauty of using universal reference? ;)

Reply:
Not "needed", but because of the way we initialize the job in the context, it will be faster with universal references (move instead of copy)